src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -367,6 +405,34 @@ public Cci.IMetadataConstant Constant
         }
     }
 
+    internal sealed class ModuleVersionIdField : SynthesizedStaticField
+    {
+        internal ModuleVersionIdField(string name, Cci.INamedTypeDefinition containingType, Cci.ITypeReference type)
```

Comment:
It looks like the name is always "MVID", consider inlining it rather than passing as a parameter.